packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_webview_controller.dart

```diff
@@ -277,6 +277,36 @@ abstract class PlatformWebViewController extends PlatformInterface {
       'getUserAgent is not implemented on the current platform',
     );
   }
+
+  /// Sets a callback that notifies the host application that the web page
+  /// wants to display a JavaScript alert() dialog.
+  Future<void> setJavaScriptAlertDialogCallback(
+      Future<void> Function(String message)
+          javaScriptAlertDialogCallback) async {
```

Comment:
Since this is only setting one callback method, the pattern should be 
```suggestion
  Future<void> setOnJavaScriptAlertDialog(
      Future<void> Function(String message)
          onJavaScriptAlertDialog) async {
```

Same for the two methods below.